app/services/hyrax/collection_types/permissions_service.rb

```diff
@@ -159,7 +159,7 @@ def self.create_access_for_collection_type?(collection_type:, user: nil, ability
       #   If calling from Abilities, pass the ability.  If you try to get the ability from the user, you end up in an infinit loop.
       def self.access_to_collection_type?(collection_type:, access:, user: nil, ability: nil) # rubocop:disable Metrics/CyclomaticComplexity
         return false unless user.present? || ability.present?
-        return false unless user && collection_type
+        return false unless collection_type
```

Comment:
was this just a bug?

what will happen on `user` references below? should we consider `user ||= ability&.current_user`?

i realize that whatever complex logic is happening here predates these changes, but i want to understand at least a little, if possible. :)